.github/workflows/jdk-ea-stable.yml

```diff
@@ -1,5 +1,5 @@
 
-name: avaje-jsonb EA stable
+name: JDK EA Stable
```

Comment:
I didn't want this renamed. It is actually like that for a reason.

Reply:
what is the reason?

Reply:
I'd put all the EA build badges across all the avaje projects together in the one place. One place to see the status of each and every EA build. I did that for Ebean ORM.

Reply:
Where is this place?

Reply:
It doesn't exist yet for avaje ... but it might be 2 places. One for avaje and another for the Oracle Early Access programme - but neither of these exist yet (but Ebean ORM has one and it's useful there).